performance/SwiftUIPerformanceQuickstart/PerformanceExample/Shared/UITests.swift

```diff
@@ -64,79 +74,116 @@ class UITests: XCTestCase {
     #endif
   }
 
+  func download(image: Bool = false) throws {
+    #if os(iOS)
+      XCTAssert(download.button.isHittable, "Reached invalid state.")
+      download.button.tap()
+    #else
+      if !image {
+        XCUIRemote.shared.press(.up)
+        XCUIRemote.shared.press(.up)
+        sleep(1)
+        XCTAssert(download.cell.hasFocus, "Reached invalid state.")
+      }
+      XCUIRemote.shared.press(.select)
+    #endif
+  }
+
   func classify(image: Bool = false) throws {
     #if os(iOS)
-      XCTAssert(app.buttons["Classify Image"].isHittable, "Reached invalid state.")
-      app.buttons["Classify Image"].tap()
+      XCTAssert(classify.button.isHittable, "Reached invalid state.")
+      classify.button.tap()
     #else
```

Comment:
Please explicitly check for tvos and do a `fatal_error` in the `#else`

Reply:
In line 25 of `UITests.swift`, I add the following logic runs before any test is set to run, which is why I didn't include `fatalError` or explicit checks for `tvOS` throughout...
```
  override func setUpWithError() throws {
    #if !os(iOS) && !os(tvOS)
      throw Error("Unsupported platform.")
    #endif
```